sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_additional_includes.py

```diff
@@ -68,7 +71,15 @@ def _copy(src: Path, dst: Path) -> None:
         if src.is_file():
             _general_copy(src, dst)
         else:
-            shutil.copytree(src, dst, ignore=shutil.ignore_patterns("__pycache__"))
+            # use os.walk to replace shutil.copytree, which may raise FileExistsError
+            # for same folder, the expected behavior is merging
+            # ignore will be applied during upload
+            for root, _, files in os.walk(src):
+                dst_root = Path(dst) / Path(root).relative_to(src)
+                if not dst_root.is_dir():
+                    dst_root.mkdir()
+                for name in files:
+                    _general_copy(Path(root) / name, dst_root / name)
```

Comment:
what do you think about using IgnoreFile in _general_copy?

Reply:
similar for me, but `_general_copy` is not so ideal as it's for single file
I have just pushed one version that apply ignore file in this function `_copy`, you may take a look